### ISSUE TITLE
FaceTools not working

### ISSUE THREAD

**Issue by sheeba46** [ymgenesis/FaceTools#5]
Hi, I have had no joy in getting any of the nodeutiliies to work. I thought you installation instructions were very clear but still I do not have any success. I am on Windows 11 and InvokeAI 3.3. 

If you have the time could you suggest where I am going wrong?

Cheers

Bill G
Australia

**Comment by Mikec78660** [comment]
Yea, I'm getting 

'RemoveBackgroundInvocation' object has no attribute 'workflow'

Maybe they are not compatible with  3.4 ?

**Comment by ymgenesis** [comment]
Correct, I haven’t updated them recently. On Nov 15, 2023, at 2:24 PM, Mikec78660 ***@***.***> wrote:﻿
Yea, I'm getting
'RemoveBackgroundInvocation' object has no attribute 'workflow'
Maybe they are not compatible with  3.4 ?
